WPMedia/fusion-news-theme-blocks
# THEMES-1022 - fix: gallery aspect radio

## Description

The image of the gallery maintains its aspect ratio and displays black bars.

## Jira Ticket

- [THEMES-1022](https://arcpublishing.atlassian.net/browse/THEMES-1022)

## Acceptance Criteria

Looks like https://corecomponents.arcpublishing.com/pagebuilder/editor/curate?p=p5wqrYWA6qfN4jt0t&v=vcMqML8L8fN4jt0t

## Test Steps

1. Checkout this branch `git checkout THEMES-1022`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/gallery-block`
3. Open the Homepage in the page editor and check the Gallery Block Images in mobile and desktop.

## Effect Of Changes

### Before

Images don't maintain aspect ratio and display black bars in gallery block.

### After

Images maintain aspect ratio and display black bars in the gallery block 

## Dependencies or Side Effects

- Feature pack pr for local development:

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1022]: https://arcpublishing.atlassian.net/browse/THEMES-1022?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
